docs/source/inference/bootstrap_ci.rst

```diff
@@ -0,0 +1,79 @@
+Bootstrap Confidence Intervals
+********************************
+We use the notation and formulations provided in chapter 10 of :cite:`Hansen2020`.
+
+Bruce E. Hansen - Econometrics,  (https://www.ssc.wisc.edu/~bhansen/econometrics)
+
+The first supported confidence interval type is the **"percentile"** confidence
+interval, as discussed in section 10.10 of the Hansen textbook.
+Let :math:`\{ \hat{\theta}_1^*, ..., \hat{\theta}_B^*\}` denote the estimates of
+estimator :math:`\hat{\theta}` for the B bootstrap samples. The idea of the percentile
+confidence interval is to simply take the empirical quantiles :math:`q_{p}^*` of
+this distributions, so we have
+
+.. math:: CI^{percentile} = [q_{\alpha/2}^*, q_{1-\alpha/2}^*].
+
+The second supported confidence interval **"normal"** is based on a normal approximation
+and discussed in Hansen's section 10.9.
+Let :math:`s_{boot}` be the sample standard error of the distribution
+of bootstrap estimators, :math:`z_q` the q-quantile of a standard normal
+distribution and :math:`\hat{\theta}` be the full sample estimate of :math:`\theta`.
+Then, the asymptotic normal confidence interval is given by
+
+.. math:: CI^{normal} = [\hat{\theta} - z_{1- \alpha/2} s_{boot},  \hat{\theta} + z_{1- \alpha/2} s_{boot}].
```

Comment:
Being nitpicky :). Can you wrap formulas at 88 characters as well? doc8 does currently not raise an error for exceeding lines in directives, but it will probabably do it in the future. For longer lines use a block instead of a single line for readibility.

```rst
.. math::

    a + b = c
    d + e = f
```

Reply:
Done in #202